include/swift/Frontend/Frontend.h

```diff
@@ -533,6 +537,14 @@ class CompilerInstance {
   /// i.e. if it can be found.
   bool canImportSwiftConcurrency() const;
 
+  /// Verify that if an implicit import of the `StringProcessing` module if
+  /// expected, it can actually be imported. Emit a warning, otherwise.
+  void verifyImplicitStringProcessingImport();
```

Comment:
This doesn't seem to be called, is that expected?

Reply:
Good catch. Fixed.